compositor_chromium/src/cef_ref.rs

```diff
@@ -98,14 +108,17 @@ impl<T: CefStruct> CefRefPtr<T> {
 
     extern "C" fn has_one_ref(base: *mut chromium_sys::cef_base_ref_counted_t) -> c_int {
         let self_ref = unsafe { Self::from_base(base) };
-        let is_one_ref = (self_ref.ref_count.load(Ordering::SeqCst)) == 1;
+        // `Ordering::Acquire` - we want every previous write to become visible
+        // so that we are sure there is only one reference
```

Comment:
```suggestion
        // `Ordering::Acquire` - since this is used by cef and this being true means 
        // that we have exclusive access to the data, we need to make sure that all 
        // accesses to the data finish before this function returns, in case cef wants 
        // to modify the data afterwards
```